### ISSUE TITLE
Meta: Top Five Task You Wish Were Done Today

### ISSUE THREAD

**Issue by QuantamHD** [The-OpenROAD-Project/OpenROAD#1758]
What are the top five tasks you wish were done today? i.e. 5 most impactful tasks for the project

**Comment by maliberty** [comment]
My list
- Restructuring (rmp)
- Post-drt final fixing flow which will require incremental detailed routing.  This would replace the 'eco' flow in OL.
- Turn on the post-grt fixing flow in ORFS
- Enable RTL-MP and hierarchical synthesis with selective flattening
- Improved CTS (better clustering, GH-tree, improved buffer selection, smart NDRs)

**Comment by QuantamHD** [comment]
@antonblanchard @arlpetergadfort Would love to hear from you as well

**Comment by arlpetergadfort** [comment]
My list:

- CTS, the tree built something makes fixing setup/hold worse by poorly selecting buffers, improving CTS would help with that I think.
- Resizer setup repair gives up too soon when fixing setup violations, that could be improved with a better stop criteria, etc.
- Detail placement allowing specification of minimum gap (not just padding).

I'm sure there are more, but these are the ones at the front of my mind at the moment.

**Comment by arlpetergadfort** [comment]
> @arlpetergadfort is the minimum gap to deal with the lack of 1 site filler in some pdks or something else?

@maliberty yes, exactly. Without it increasing density would become extremely difficult.

**Comment by antonblanchard** [comment]
- +1 for "Post-drt final fixing flow which will require incremental detailed routing."
- +1 for "Improved CTS"
- VHDL/GHDL support. This is being worked on and I unfortunately haven't had time to test and offer feedback yet.

**Comment by maliberty** [comment]
stale
